src/web/handlers/FlingGestureHandler.ts

```diff
@@ -32,6 +32,10 @@ export default class FlingGestureHandler extends GestureHandler {
 
     this.enabled = enabled;
 
+    if (this.config.shouldCancelWhenOutside !== undefined) {
+      this.setShouldCancelWhenOutside(this.config.shouldCancelWhenOutside);
+    }
```

Comment:
This is repeated for every handler (I believe Pan already had it), cannot it be moved to the base class?